code/game/machinery/virtual_reality/ar_console.dm

```diff
@@ -0,0 +1,135 @@
+/*
+ * This file contains the alien mind-transfer pod, or 'Alien Reality' pod.
+ */
+
+
+/obj/machinery/vr_sleeper/alien
+	name = "strange pod"
+	desc = "A strange machine with what appears to be a comfortable, if quite vertical, bed. Numerous mechanical cylinders dot the ceiling, their purpose uncertain."
+	icon = 'icons/obj/Cryogenic2.dmi'
+	icon_state = "alienpod_0"
+	base_state = "alienpod_"
+
+	eject_dead = FALSE
+	var/mirror_first_occupant = TRUE	// Do we force the newly produced body to look like the occupant?
```

Comment:
Could these vars not go on the base VR sleeper? Might be helpful/reduce copypasta.

Reply:
Mirror, probably, but the species is explicitly only ever Virtual Reality Avatar for the VR pods, the others just exist to get icons from IIRC.